recipe/meta.yaml

```diff
@@ -58,3 +50,4 @@ extra:
     - ocefpaf
     - pelson
     - gillins
+    - msarahan
```

Comment:
@msarahan I pulled the changes from https://github.com/AnacondaRecipes/geos-feedstock now to do the right thing instead of a hackish copy-n-paste :smile:  

Can you take a quick look at the recipe and the build scripts to see if this is OK?

Reply:
everything looks sane to me.  Curious to see how the CI's come out.